packages/tokens/package.json

```
@@ -16,26 +16,28 @@
   },
   "license": "MPL-2.0",
   "author": "HashiCorp Design Systems <design-systems@hashicorp.com>",
+  "type": "module",
   "scripts": {
     "typecheck": "yarn tsc --noEmit",
     "lint": "yarn eslint --quiet --ext .js,.ts",
-    "build": "ts-node --transpile-only ./scripts/build"
+    "build": "tsx ./scripts/build"
   },
   "devDependencies": {
     "@types/fs-extra": "^11.0.3",
-    "@types/lodash": "^4.14.200",
+    "@types/lodash-es": "^4",
     "@types/node": "^22.8.7",
     "@types/tinycolor2": "^1.4.5",
     "@typescript-eslint/eslint-plugin": "^8.5.0",
     "@typescript-eslint/parser": "^8.5.0",
     "del": "^5.1.0",
     "eslint": "^8.57.0",
     "fs-extra": "^11.1.1",
-    "lodash": "^4.17.21",
+    "lodash-es": "^4.17.21",
     "path": "^0.12.7",
-    "style-dictionary": "^3.9.0",
+    "style-dictionary": "^4.2.0",
```

style-dictionary now requires `prettier`. we already have it in the monorepo as it's used on all other packages, but it would be great to explicitly add it to `devDependencies` so we get rid of the yarn warning.
```bash
➤ YN0002: │ @hashicorp/design-system-tokens@workspace:packages/tokens doesn't provide prettier (pebf08), requested by style-dictionary.
```

I added an in-line suggestion, but it will require a `yarn.lock` update too.
```suggestion
    "prettier": "^3.3.2",
    "style-dictionary": "^4.2.0",
```